star: readme file (#2278)
graphile/crystal: utils/README.md

# Graphile\*

[![GitHub Sponsors](https://img.shields.io/github/sponsors/benjie?color=%23ea4aaa&label=github%20sponsors)](https://github.com/sponsors/benjie)
[![Discord chat room](https://img.shields.io/discord/489127045289476126.svg)](http://discord.gg/graphile)
[![Mastodon Follow](https://img.shields.io/mastodon/follow/109982006777922021?domain=https%3A%2F%2Ffosstodon.org&style=flat&label=mastodon&color=%236364FF)](https://fosstodon.org/@graphile)

> [![Static Badge](https://img.shields.io/badge/bsky-graphile.org-blue?style=flat&color=%230285FF)](https://bsky.app/profile/graphile.org)

Reviewer: ```suggestion
[![Bluesky](https://img.shields.io/badge/bsky-graphile.org-blue?style=flat&color=%230285FF)](https://bsky.app/profile/graphile.org)
```

_**Graphile\* - the stars we forge to build our universe**_

**Documentation**: https://star.graphile.org

## About

> A collection of useful utilities we built whilst working on Graphile suite of

Author: ```suggestion
A collection of useful utilities we built whilst working on the Graphile suite of
```

> A collection of useful utilities which work in tandem with the rest of the
> Graphile suite of open source developer tools.

Reviewer: ```suggestion
A collection of useful utilities we built whilst working on Graphile suite of
open source developer tools, that may well be useful for your projects too.
```
